packages/v4/patternfly-docs/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -5,45 +5,78 @@ section: components
+Be aware that anytime the user navigates aways from the wizard, there is a risk that any information entered up until that point will be lost. At a minimum, we recommend opening a modal alert when the user tries to do this warning of potential data loss and confirming that they want to leave the wizard. Optionally you should consider including a "Save as draft" feature that will allow work in progress to be saved and recalled upon returning to the wizard.
 
-### Simple wizard
-<img src="./img/standard-wizard.png" alt="Simple wizard" />
+### Progressive wizard
+A progressive wizard takes the same form as the standard wizard or wizard with sub-steps above but accommodates a situation where the exact number of steps is not known when the wizard is first loaded. Steps in the sidebar can be changed or added as the user progresses through the wizard.
 
+A walk-through of a progressive wizard might look like this:
+
+<img src="./img/wizard-progressive-step1.png" alt="Wizard progressive step 1" />
 
-1. **Header (modal wizard only):** Modal wizards always have a header area that minimally contains the wizard title and a close button. The title should be action oriented and reflect the intended outcome of the wizard, such as "Create resource." You may optionally also include a description below the wizard title.
-2. **Steps sidebar:** Enumerated steps are displayed in the sidebar. Steps can be fixed or updated as the user proceeds through the process (see progressive wizard).
-3. **Visited step:** A step that has been already visited. In most cases users can click this step to return to a prior point in the flow.
-4. **Current step:** The current step is always highlighted.
-5. **Disabled step:** Steps that follow the current step are usually disabled to enforce the sequential flow.
-6. **Step title:** The Step title is unique to each step, and should reinforce the outcome of the individual step. It may match the Current step indicated in the steps sidebar, or as in the example image, it may be more verbose.
-7. **Body:** The contents of a step itself. Any valid form elements can be incorporated within the body. The size of the modal that contains the wizard can be adjusted based on the contents of the body section. If the height of the body section causes the bottom of the modal to push beyond the height of the viewport, a vertical scrollbar will appear. However it is recommended to break your workflow into small enough steps so that scrolling is not necessary on typical monitor sizes.
-8. **Button footer:** Buttons control the wizard flow. Default buttons are "Next," "Back," and "Cancel." You may optionally add other actions, such as "Skip to finish" or "Start over." There should only be one primary action in the footer and it should be the farthest left action.
+
+Step 1: The user is presented with a Get started screen where they can specify what they want to do.
```

Comment:
Could you bold "Step 1" "Step 2" "Step 3" - like just the words before the colon